client/lib/sites-list/test/test.js

```diff
@@ -1,104 +1,110 @@
 			} );
-
 		} );
 
-		it( 'should set attributes properly', function() {
-			var sitesList = SitesList(),
-				site, origSite;
-			sitesList.initialize( original.slice() );
-
-			site = sitesList.get()[0];
-			origSite = original[0];
-
-			for ( var prop in origSite ) {
-				assert( site[ prop ] ===  origSite[ prop ] );
+		it( 'should set attributes properly', () => {
+			let site = initializedSites[ 0 ];
+			let origSite = originalData[ 0 ];
 
-			}
+			forEach( origSite, ( value, prop ) => {
+				assert.deepEqual( value, site[ prop ] );
+			} );
 		} );
 
-		it( 'should add change handlers', function() {
-			var sitesList = SitesList();
-
-			sitesList.initialize( original.slice() );
-			assert( sitesList.get().length, original.length );
-			sitesList.get().forEach( function( site ) {
-				assert( site.listeners( 'change' ) );
+		it( 'should add change handlers', () => {
+			forEach( initializedSites, ( site ) => {
+				assert.isDefined( site.listeners( 'change' ) );
 			} );
-
 		} );
 	} );
 
-	describe( 'updating', function() {
-		it( 'updating should not create new Site instances', function() {
-			var sitesList = SitesList(),
-				originalList, updatedList;
+	describe( 'updating', () => {
+		let sitesList, originalData, updatedData, originalList;
 
-			sitesList.initialize( original.slice() );
+		beforeEach( () => {
+			originalData = cloneDeep( data.original );
+			updatedData = cloneDeep( data.updated );
+			sitesList = SitesList();
+			originalList = sitesList.initialize( originalData );
+		} );
 
-			originalList = sitesList.get();
-			sitesList.update( updated.slice() );
-			updatedList = sitesList.get();
+		it( 'updating should not create new Site instances', () => {
+			sitesList.update( updatedData );
+			let updatedList = sitesList.get();
 
-			originalList.forEach( function( site, index ) {
-				assert( site === updatedList[index ] );
+			forEach( originalList, ( site, index ) => {
+				assert.strictEqual( site, updatedList[ index ] );
 			} );
 		} );
 
-		it( 'should update attributes properly', function() {
-			var sitesList = SitesList(),
-				site, updatedSite;
-			sitesList.initialize( original.slice() );
-
-			sitesList.update( updated.slice() );
+		it( 'should update attributes properly', () => {
+			sitesList.update( updatedData );
 
-			site = sitesList.get()[0];
-			updatedSite = updated[0];
+			let site = sitesList.get()[ 0 ];
+			let updatedSite = updatedData[ 0 ];
 
-			for ( var prop in updatedSite ) {
-				assert( site[ prop ] ===  updatedSite[ prop ] );
-
-			}
+			forEach( updatedSite, ( updatedValue, prop ) => {
+				assert.deepEqual( updatedValue, site[ prop ] );
+			} );
 		} );
-
-
 	} );
 
-	describe( 'change propagation', function() {
-		it( 'should trigger change when site is updated', function() {
-			var sitesList = SitesList(),
-				site, called = false;
+	describe( 'change propagation', () => {
+		it( 'should trigger change when site is updated', () => {
+			let sitesList = SitesList();
```

Comment:
Beside last line we should use const, and we also use one `const`/`let` expression for several declarations. Seehttps://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#declaring-variables-with-var.